.monorepo.json

```diff
@@ -87,6 +87,11 @@
       "hash": "9d2161b7ca9044c2286c7d66e10ead12",
       "snapshotRepo": "angular/angular-devkit-architect-cli-builds"
     },
+    "@angular-devkit/benchmark": {
```

Comment:
Is it a necessary part of this PR to introduce (and publish) a new npm package?

Reply:
It was part of the original design, so that the benchmark could be ran over other projects.

Reply:
I'm not sure we should publish it yet though.  There's already a large amount of packages under management.

Reply:
How about we build it, but exclude from the publishing process?

Reply:
sounds good. is there a facility to do that?

Reply:
The publish script honors the `"private": true,` field in package.json and does not attempt to publish those packages. I added that to the benchmark package.